packages/css/src/sort.ts

```diff
@@ -8,14 +8,22 @@ import { sortAtomicStyleSheet } from './plugins/sort-atomic-style-sheet';
  * Sorts an atomic style sheet.
  *
  * @param stylesheet
- * @returns
+ * @returns the sorted stylesheet
  */
 export function sort(
   stylesheet: string,
   {
     sortAtRulesEnabled,
     sortShorthandEnabled,
-  }: { sortAtRulesEnabled: boolean | undefined; sortShorthandEnabled: boolean | undefined }
+  }: { sortAtRulesEnabled: boolean | undefined; sortShorthandEnabled: boolean | undefined } = {
```

Comment:
Given this is now a required prop, I actually expect this to fail in TS anywhere `sort()` is called…but yeah I guess we have to find them directly in code if we find them:
```dif
-sort(styles);
+sort(styles, { sortAtRulesEnabled: true, sortShorthandEnabled: true });
```

Not the best configuration experience, but we'll survive for now.

Reply:
![Screenshot 2024-10-23 at 16 54 31](https://github.com/user-attachments/assets/f3fd0bc6-406b-4e89-ac5f-11b65e54c73b)

typescript says it's optional 🤔

Reply:
but yeah we would need to turn on `sortShorthandEnabled` manually everywhere in order to roll it out

(how would we make sure we don't forget? 🤔)